### ISSUE TITLE
Add simple feedback mechanism

### ISSUE THREAD

**Issue by brendangadd** [StatCan/StatCan.OrchardCore#14]
As a user, I want to leave general feedback via the web portal to share my experience with the developers.

A simple up/down rating with a comment box is probably sufficient. We could reuse the GitHub module to submit an issue to a support or feedback repository with the content. This way we could automatically track progress on feedback that needs action.

Open to other suggestions.

**Comment by bbronzan** [comment]
It's just about done. Here's how it works:

1. Choose your mood
2. Enter your feedback
3. Send!

![image](https://user-images.githubusercontent.com/62296192/82686186-6de90e80-9c23-11ea-80dc-915d13176f65.png)

The feedback will be sent to our Slack workspace and our GitHub so we can monitor the virtual hub and virtual labs the same way we do for our website and other products.

Since you want a copy of the feedback as well:

- provide us the GitHub repo and bot token (if it's different than the request namespace token)
- in the short term we might have to manually add these issues since the Orchard workflow can only handle one token
- once Orchard is upgraded (currently being done) we can forward  a copy automatically to your repo

**Comment by jptissot** [comment]
@M-A-Boucher Another thing we could do for the bot token, we could create a bot account and they could give it access to write issues

**Comment by brendangadd** [comment]
This is looking really great, and I love that things are getting piped into both locations. Private channel in Slack, right? Ultimately, with link to GitHub issue?

> How are you feeling and what could be improved

For me, this text splits the feedback form into two distinct parts:
* How are you feeling: choose a face
* What could be improved: comment box

In terms of the mental space we put people in, I would like to see the two elements tied together (e.g. "How are you feeling? Comments?" or even just "How was your experience?") – kind of like Steam reviews where people are writing whatever they want to support their thumbs up/down.

**Comment by bbronzan** [comment]
Good call. It was way too wordy, so I changed it to "How was your experience?"

**Comment by M-A-Boucher** [comment]
> @M-A-Boucher Another thing we could do for the bot token, we could create a bot account and they could give it access to write issues

Good call. Our bot will be the feedback messenger for any repo that wants feedback routed to them.

**Comment by bbronzan** [comment]
As per: https://github.com/StatCan/StatCan.OrchardCore/issues/24

Instead of GitHub, the DAaaS folks decided to go with a Slack channel endpoint for the feedback.

Integrated and tested. Good to go.
